.travis.yml

```diff
@@ -7,12 +7,15 @@ python:
 sudo: false
 
 env:
-    - DJANGO=1.5.12
-    - DJANGO=1.6.10
+    - DJANGO=1.5.12 POSTGRES_TEST_USER=postgres POSTGRES_TEST_NAME=avocado MYSQL_TEST_USER=root MYSQL_TEST_NAME=avocado
```

Comment:
I too am uncomfortable with it, but to make the migration path a bit easier in case Django 1.4 is being used, I decided that 2.4 will still support Django 1.5. Harvest 2.5 will drop 1.5 which will hopefully follow in a relatively short timespan.